src/Export/Adapters/SalesFrequencyAdapter.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+declare(strict_types=1);
+
+namespace FINDOLOGIC\FinSearch\Export\Adapters;
+
+use DateTimeImmutable;
+use FINDOLOGIC\Export\Data\SalesFrequency;
+use Shopware\Core\Content\Product\ProductEntity;
+use Shopware\Core\Defaults;
+use Shopware\Core\Framework\DataAbstractionLayer\EntityRepository;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Criteria;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\EqualsFilter;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\MultiFilter;
+use Shopware\Core\Framework\DataAbstractionLayer\Search\Filter\RangeFilter;
+use Shopware\Core\System\SalesChannel\SalesChannelContext;
+
+class SalesFrequencyAdapter
+{
+    /** @var EntityRepository $orderLineItemRepository */
+    private $orderLineItemRepository;
+
+    /** @var SalesChannelContext $salesChannelContext */
+    private $salesChannelContext;
+
+    public function __construct(
+        EntityRepository $orderLineItemRepository,
+        SalesChannelContext $salesChannelContext
+    ) {
+        $this->orderLineItemRepository = $orderLineItemRepository;
+        $this->salesChannelContext = $salesChannelContext;
+    }
+
+    public function adapt(ProductEntity $product): ?SalesFrequency
+    {
+        $orders = $this->orderLineItemRepository->searchIds(
+            $this->getCriteria($product),
+            $this->salesChannelContext->getContext()
+        );
+
+        if (!$orders->getTotal()) {
```

Comment:
This would also fail if it was `0` and `getTotal` always returns an `int` anyway, so what's the point of this check?